docs/response-parameters.rst

```diff
@@ -1,21 +1,29 @@
 Response Parameters
 ===================
 
-Descriptions of the custom response parameters that are marked with a leading ``@`` for the 
-data extraction and contributions endpoints.
+Description of the response parameters.
 
 General Parameters
 ------------------
 
-* ``@osmId`` - id of the OSM feature with its type (node/way/relation)
-* ``@version`` - version of the feature
-* ``@changesetId``- id assigned to a changeset
-* ``@osmType``- type (node/way/relation) of OSM feature
+Description of parameters that are present in every response.
+
+* ``attribution`` - copyrights and attribution
+* ``apiVersion`` - version of the ohsome API
 
+Aggregation Parameters
+----------------------
 
 Extraction Parameters
 ---------------------
 
+Descriptions of the custom response parameters that are marked with a leading ``@``.
+
+* ``@osmId`` - id of the OSM feature with its type (node/way/relation)
```

Comment:
IMHO we should be more consistent with wordings here. Since the API endpoints use the term _elements_, this should be more appropriate:

```suggestion
* ``@osmId`` - id of the OSM element, including its type (e.g. node/1)
```